java/src/com.zeroc.ice/src/main/java/com/zeroc/Ice/Current.java

```diff
@@ -213,11 +210,8 @@ private OutgoingResponse createOutgoingResponseCore(Throwable exc) {
         OutputStream ostr;
 
         if (requestId != 0) {
-            // The default class format doesn't matter since we always encode user exceptions in
-            // Sliced format.;
```

Comment:
In addition to moving the comment onto a single line, I also removed this random semicolon from it.